content/docs/started/k8s/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Kubeflow on Existing Kubernetes Clusters"
-description = "Instructions for installing Kubeflow on your existing clusters"
+title = "Kubernetes Install"
```

Comment:
Revert to the original title. The new one isn't clear or specific enough. It sounds like a guide to installing K8s.